src/sample/Game.java

```diff
@@ -53,7 +53,7 @@ void executeAutoMove() {
         // make next move
         ++turnCount;
         System.out.println("Current turn: " + turnCount);
-        Bot threadBot = this.bot;
+        // Bot threadBot = this.bot;
```

Comment:
If this isn't needed, we can just remove it.

Reply:
I am trying to implement threads so anything to do with those right now are not in use but I did not want to all out delete them. In this case commenting is legit so this fine.

Reply:
Just don't want to kill it since this is where it will be used when the threading is implemented